### PR TITLE
use yarn and remove lerna, update some deps

### PR DESCRIPTION
**Pull request by rdinicut**
The package.lock was messed up and audit fix was breaking the project. While lerna has some benefits it has some bugs it seems. yarn workspaces work just fine and it fixed all the security issues

Had to regenerate snapshots because of typescript update. Could not do it to 4 because of the centrifuge node api files.

### CODE REVIEW

File: packages/server/src/contacts/__tests__/contacts.controller.spec.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ describe('ContactsController', () => {
           name: 'Joe',
         } as Contact);
       } catch (err) {
-        expect(err.message.message).toEqual('Contact address must have ETH format');
+        expect(err.message.message).toEqual('This method only supports 0x-prefixed hex strings but input was: undefined');
```
Comment (charleenfei) [inline packages/server/src/contacts/__tests__/contacts.controller.spec.ts:98]: why is this changed? i think we should have ETH formatted addresses even in the tests, there is a new test for this

File: packages/server/src/users/users.controller.ts [hunk 2]
```diff
@@ -144,7 +144,7 @@ export class UsersController {
           // @ts-ignore
           config.admin.chain,
       );
-      user.account = account.identity_id.toLowerCase();
+      user.account = account.identityId.toLowerCase();
```
Comment (charleenfei) [inline packages/server/src/users/users.controller.ts]: i am pretty sure this will fail when we actually try to call on the node, we need to use identity_id bc swagger generates the wrong param names for the api

**Review by charleenfei — CHANGES_REQUESTED** [review]

File: packages/server/src/users/users.controller.ts (hunk 2 above)
Comment (rdinicut) [reply packages/server/src/users/users.controller.ts]: Ok, I i will ts ignore it then

File: packages/server/src/contacts/__tests__/contacts.controller.spec.ts (hunk 1 above)
Comment (rdinicut) [reply packages/server/src/contacts/__tests__/contacts.controller.spec.ts:98]: Because of the update on eth util lib. It will throw that error in case it is not a good format

**Review by charleenfei — APPROVED** [review]
